src/server/handlers/security-handler.ts

```diff
@@ -0,0 +1,11 @@
+import type { Response } from "express";
+
+export default async ({ res }: { res: Response }) => {
+  res.setHeader("content-type", "text/plain; charset=utf-8");
+
+  res.send(`Contact: mailto:security@lemmy.ml
+  Contact: mailto:admin@` + process.env.LEMMY_UI_HOST + `
```

Comment:
`LEMMY_UI_LEMMY_EXTERNAL_HOST` is the one with the actual domain.tld . The other ones are for the internal isomorphic server.

Reply:
Done